ql/src/java/org/apache/hadoop/hive/ql/optimizer/GroupByOptimizer.java

```diff
@@ -28,19 +28,11 @@
 import java.util.Set;
 import java.util.Stack;
 
+import org.apache.hadoop.hive.ql.exec.*;
```

Comment:
AFAIK Hive code style doesn't allow wild card imports.
Here is some discussion why https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Please revert this change.

Reply:
My bad, I missed this.